rdt/transformers/base.py

```diff
@@ -165,6 +165,32 @@ def _build_output_columns(self, data):
             self.column_prefix += '#'
             self.output_columns = list(self.get_output_types().keys())
 
+    def __str__(self):
+        """Represent initialization of transformer as text.
+
+        Returns:
+            str:
+                The name of the transformer followed by any non-default parameters.
+        """
+        class_name = self.__class__.__name__
+        args = []
+        default = self.__class__()
+        for arg, value in self.__dict__.items():
+            if default.__dict__[arg] != value:
+                args.append(f'{arg}={value}')
```

Comment:
I think that we should add the following:
```python
if isinstance(value, str):
    value = f"'{value}'"
```
The reason why is that currently with this implementation if I do the following:
```python
In [1]: from rdt.transformers import FloatFormatter

In [2]: trans = FloatFormatter(missing_value_replacement='mean')

In [3]: print(trans)
FloatFormatter(missing_value_replacement=mean)
```
Where I think the expected result should be `FloatFormatter(missing_value_replacement='mean')`